grantpicks/serializers.py

```
@@ -0,0 +1,347 @@
+    project_1 = AccountSerializer()
+    project_2 = AccountSerializer()
+    voted_project = AccountSerializer()
+
+    class Meta:
+        model = VotePair
+        fields = [
+            'pair_id',
+            'project_1',
+            'project_2',
+            'voted_project',
+        ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)